server/test/list_objects.go

```diff
@@ -203,33 +338,149 @@ func runListObjectsTests(t *testing.T, ctx context.Context, testCases []listObje
 	}
 }
 
-func setupTestListObjects(store string, datastore storage.OpenFGADatastore) (context.Context, storage.OpenFGADatastore, string, error) {
+// Used to avoid compiler optimizations (see https://dave.cheney.net/2013/06/30/how-to-write-benchmarks-in-go)
+var listObjectsResponse *openfgapb.ListObjectsResponse //nolint
+
+func BenchmarkListObjectsWithReverseExpand(b *testing.B, ds storage.OpenFGADatastore) {
+
 	ctx := context.Background()
-	data, err := os.ReadFile(gitHubTestDataFile)
-	if err != nil {
-		return nil, nil, "", err
+	store := ulid.Make().String()
+
+	model := &openfgapb.AuthorizationModel{
+		Id:            ulid.Make().String(),
+		SchemaVersion: typesystem.SchemaVersion1_0,
```

Comment:
This is 1.1 🤔  why didn't the WriteAuthModel fail?

Reply:
The datastore doesn't have validation logic like that, so that's why it passed. I'll change this to `typesystem.SchemaVersion1_1` though. Good catch